### PR TITLE
Make resample documentation better

### PR DESCRIPTION
**Pull request by anshbansal**
I removed the checkboxes because they were not relevant. This is a docstring update. I found the answer in https://stackoverflow.com/questions/17001389/pandas-resample-documentation but it should have been present in the docs

### CODE REVIEW

File: pandas/core/generic.py [hunk 1]
```diff
@@ -7708,6 +7708,8 @@ def resample(
         ----------
         rule : DateOffset, Timedelta or str
             The offset string or object representing target conversion.
+            The str value refers to frequency string.
```
Comment (jreback) [inline pandas/core/generic.py]: this line is redundant. you can add a link the documentation if you'd like; i think we have this link elsewhere.

**Comment by datapythonista** [comment]
@anshbansal still interested on finishing this? If so, please remove the line that doesn't add value and create a link to the mentioned page/section. Thanks

**Comment by anshbansal** [comment]
@jreback  @datapythonista Can you tell me what link is being referred to in the previous comment?

File: pandas/core/generic.py [hunk 2]
```diff
@@ -7708,6 +7708,8 @@ def resample(
...
+            The list of valid values can be found in the timeseries user guide.
```
Comment (datapythonista) [inline pandas/core/generic.py]: @anshbansal making this a link to the `timeseries user guide`, besides saying it.

**Comment by datapythonista** [comment]
Closing as stale, @anshbansal let us know if you'd like to finish this.

File: pandas/core/generic.py (hunk 2 above)
Comment (oaustegard) [reply pandas/core/generic.py:7712]: Where, pray tell - [other than on SO](https://stackoverflow.com/a/17001474/27938) - would one find this list of valid values?

**Pull request closed without merging**